src/main/drivers/mcu/stm32/pwm_output.c

```diff
@@ -35,6 +35,10 @@
 
 #include "pg/motor.h"
 
+#ifndef PWM_SHUTDOWN_DELAY
+#define PWM_SHUTDOWN_DELAY 600
```

Comment:
Adding the unit at the end makes it a bit clearer
```suggestion
#ifndef PWM_SHUTDOWN_DELAY_MS
#define PWM_SHUTDOWN_DELAY_MS 600
```

Reply:
Thank you :)